src/api/liskApi.js

```diff
@@ -284,26 +284,26 @@ LiskAPI.prototype.listStandbyDelegates = privateApi.wrapSendRequest(GET, 'delega
 LiskAPI.prototype.searchDelegateByUsername = privateApi.wrapSendRequest(GET, 'delegates/search', username => ({ username }));
```

Comment:
I prefer to keep it `searchDelegateByUsername`.

Because it does not give a single response, even when a delegate name is exactly matched, it will display others that also contain the same strings in their delegate name.